cinema_service/settings.py

```diff
@@ -86,8 +92,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ["POSTGRES_DB"],
```

Comment:
It is good that you are using environment variables to store your database settings. This is a good practice for keeping sensitive data out of your code.